assets/src/components/Line.js

```diff
@@ -17,9 +17,9 @@ function Line (props) {
   //   })
   // })
 
```

Comment:
Can you remove the comment from L13-L18? I know you did not make that change and was skipped in the previous PR might as well clean that up since that not huge effort

Reply:
done